src/public/components/CameraPage.js

```diff
@@ -120,7 +123,7 @@ export default class CameraPage extends React.Component {
         <div>
           <button className="screenShotButton" onClick={this.takeScreenshot}>Take Screenshot</button>
           <button className="populateAttendanceRecord" onClick={this.populateAttendanceRecord}> Populate Attendance Records </button>
-          <button className="lateStudentButton" onClick={this.sendLateEmails}>Send Email to late Students</button>
+          <button className="lateStudentButton" onClick={this.sendLateEmails(this.state.selectedTimeCutoff)}>Send Email to late Students</button>
```

Comment:
This line is causing test to fail, function being invoked when mounted instead of on click, move this.state.selectedTimeCutoff to function